test/util/common.ts

```diff
@@ -290,14 +290,21 @@ export async function advanceEVMByTime(_timeSeconds: number) {
 }
 
 // utility funciton to compare Big Numbers, expecting them to be within x%, +/-
-export function compareBN(
+export function requireBigNumberIsClose(
   actual: BigNumber,
   expected: BigNumber,
   tolerancePercent: number = 1
```

Comment:
Nit: Would suggest that we either do not have a default value here at all, or if we do keep this default value (my preference to keep, and be `1`) that we don't also pass in `1` in all the call-sites above but instead make use of the default.

Reply:
good call, updated to not pass in `1` in all the call-sites, and kept default value of `1` in commit 3159566